clients/include/testing_gemv.hpp

```diff
@@ -168,7 +168,7 @@ void testing_gemv(const Arguments& arg)
     rocblas_init<T>(hA, M, N, lda);
     rocblas_init<T>(hx, 1, dim_x, abs_incx);
 
-    if(rocblas_isnan(arg.beta))
```

Comment:
`rocblas_isnan(arg.beta)` should be left alone -- it is separate from `arg.get_beta<T>()`.

Reply:
understand I will correct it on third commit "update complex change".
Do we have to check nan for betai too??

Reply:
`rocblas_isnan(arg.beta) || rocblas_isnan(arg.betai)`. It does not matter that betai is only used during complex. It will default to 0 in non-complex cases.

Reply:
understand. update it in commit "update complex change 2".